201-machine-learning-advanced/azuredeploy.json

```diff
@@ -50,6 +75,13 @@
         "description": "Name of the storage account."
       }
     },
+    "storageLocation": {
```

Comment:
remove this param - the common pattern (by far) is that these will be co-located.